models/mart/iplanrio/sisvisa/mart_sisvisa_situacao_licenciamento_estabelecimento.sql

```
@@ -0,0 +1,160 @@
+				select EstabelecimentoId, max(DataDeValidade) as data_validade_ult_licenca_expirada
+				from {{ source('brutos_sisvisa_staging', 'HistoricoDeLicenca') }}
+				where SituacaoDaLicenca = 6
+				group by EstabelecimentoId
+				) h2 on h2.EstabelecimentoId = r.EstabelecimentoId
+),
+
+--APURAÇÃO DE VALORES COBRADOS NO ÚLTIMO REQUERIMENTO
+--FOI NECESSÁRIO SOMAR, POIS HÁ CASOS DE MAIS DE UMA COBRANÇA VÁLIDA PARA UM MESMO REQUERIMENTO
+valores_cobrados as (
+	select a.id_estabelecimento_sisvisa, q.DataInicio, q.total_cobranca_ultimo_licenciamento
+	from apuracao_licenciamento a
```

SQLFluff (4.1.0)

[error] Duplicate table alias 'a'. Table aliases should be unique.

(AL04)